SPECS/subversion/subversion.spec

```diff
@@ -1,33 +1,30 @@
 Summary:        The Apache Subversion control system
 Name:           subversion
-Version:        1.14.0
-Release:        4%{?dist}
+Version:        1.14.1
+Release:        1%{?dist}
 License:        ASL 2.0
-URL:            https://subversion.apache.org/
-Group:          Utilities/System
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
+Group:          Utilities/System
+URL:            https://subversion.apache.org/
 Source0:        https://archive.apache.org/dist/%{name}/%{name}-%{version}.tar.bz2
-Patch0:         CVE-2020-17525.patch
```

Comment:
I think the patch file is still present. Probably a good idea to remove the .patch file as well.

Reply:
Thanks for pointing this out, done